### ISSUE TITLE
Please don't scan for @ShellComponent when ssh.shell.enable=false

### ISSUE THREAD

**Issue by vienhx** [fonimus/spring-boot-ssh-shell#23]
When I set to false, I have an issue

```
***************************
APPLICATION FAILED TO START
***************************

Description:

Parameter 2 of constructor in com.vp.hotit.sshd.CommentsCommand required a bean of type 'com.github.fonimus.ssh.shell.SshShellHelper' that could not be found.
	- Bean method 'sshShellHelper' not loaded because @ConditionalOnProperty (ssh.shell.enable=true) found different value in property 'ssh.shell.enable'


Action:

Consider revisiting the entries above or defining a bean of type 'com.github.fonimus.ssh.shell.SshShellHelper' in your configuration.
```

**Comment by fonimus** [comment]
@ShellComponent implies @Component, so it will always be scanned.
What you can do is put @Lazy in the sshShellHelper autowire, or put a condition on the shell component (@ConditionalProperty with ssh.shell.enable for instance)

Please let me know if you have better idea.

**Comment by vienhx** [comment]
How about using another annotation like `@SshShellComponent` implies `@ShellComponent` with `@Conditional`?

**Comment by fonimus** [comment]
Great idea, I just pushed it, will make it available for 1.0.6. Thanks !
